packages/create-strapi-app/create-strapi-app.js

```diff
@@ -10,7 +11,7 @@ let projectName;
 
 program
   .version(packageJson.version)
-  .arguments('<directory>')
+  .arguments('[directory]')
```

Comment:
why make this optional ?

Reply:
I believe we want the possibility to create a project completely interactively without any arguments passed to the create-strapi-app/starter commands while still leaving the current functionality in place.  Providing a directory, template, and quickstart will skip the prompt.

Reply:
Ok I see, why not. 

However we need ot make sure that when providing --quickstart we don't prompt anything and fail if the directory was not provided.